babel.config.js

```diff
@@ -17,10 +17,10 @@
  */
 
 /*
- * The purpose of this file is to provide a babel config file for building and distributing Backpack components.
+ * The purpose of this file is to provide babel config for configuration and use with storybook.
  *
- * For configuration and use with storybook there is a separate config file in the .storybook/babel.config.js folder,
- * which provides configuration for running the storybook development system.
+ * For babel config used in building and distributing Backpack components, there is a separate file
+ * located in the root of the project.
```

Comment:
Is this not in the root of the project?

Reply:
Huh, yeah you are right dunno what happened here 😆  clearly a rough cut and paste job? 🙈